projects/ion/src/lib/pagination/pagination.component.spec.ts

```diff
@@ -115,6 +117,22 @@ describe('Pagination > Page sizes', () => {
         expect(within(view).getByText(`${label} / página`)).toBeVisible();
       }
     );
+
+    it.each(LOADING_STATUS)(
+      'should show items per page disabled when loading is %b',
```

Comment:
I don't think items per page should be disabled if the table is not loading. If your intention is check if items per page is enabled when loading = false, and vice-versa, you should do two different assertions.

Reply:
This test is precisely to ensure that the items per page are only disabled when loading. So it checks if 'ng-reflect-disabled' is true when loading is true and false when loading is false.